Package.swift

```diff
@@ -690,6 +690,24 @@ let package = Package(
                      "SharedTestUtilities"],
       path: "Functions/Tests/CombineUnit"
     ),
+    .testTarget(
+      name: "FunctionsUnit",
+      dependencies: ["FirebaseFunctions",
+                     "SharedTestUtilities"],
+      path: "Functions/Example/Tests/",
+      exclude: [
+        "en.lproj",
+        "Tests-Info.plist",
+      ],
```

Comment:
This was added because otherwise it would complain about not having a `defaultLocalization` for spm.

Reply:
@paulb777 Have you seen this issue before?

Reply:
Good catch @ncooke3! I think those are files that are no longer used and should be deleted.  I'll investigate.